src/WebAppDIRAC/Lib/WebHandler.py

```diff
@@ -422,18 +367,14 @@ def post(self, *args, **kwargs):
         return self.get(*args, **kwargs)
 
     def threadTask(self, method, *args, **kwargs):
-        def threadJob(*targs, **tkwargs):
-            args = targs[0]
-            disetConf = targs[1]
-            self.__disetConfig.reset()
-            self.__disetConfig.load(disetConf)
-            return method(*args, **tkwargs)
+        def threadJob(*args, **tkwargs):
+            with self._setupThreadConfig():
+                return method(*args, **kwargs)
```

Comment:
while `method` is not a reserved keyword, maybe it's a bit too generic...?

Reply:
This entire `WebHandler` class should be removed entirely: https://github.com/DIRACGrid/WebAppDIRAC/issues/658